src/Compilers/Core/Portable/Symbols/IFieldSymbol.cs

```diff
@@ -66,5 +66,12 @@ public interface IFieldSymbol : ISymbol
         /// source or metadata.
         /// </summary>
         new IFieldSymbol OriginalDefinition { get; }
+
+        /// <summary>
+        /// If this field represents a tuple element, returns a corresponding default element field.
+        /// Otherwise returns null
+        /// </summary>
+        IFieldSymbol CorrespondingTupleField { get; }
+
```

Comment:
extra newline.  Comment needs to explain what a "corresponding default element field" is :)